README.md

```diff
@@ -1 +1,101 @@
-# alerting
\ No newline at end of file
+# Alerting
+
+## Introduction
+Hypertrace is the modern Observability platform and when we talk about Observability, these are the four pillars of the Observability Engineering team’s charter:
+- Monitoring
+- Alerting/visualization
+- Distributed systems tracing infrastructure
+- Log aggregation/analytics
+
+Alerting is the entry point for most of the debugging workflows and all the modern DevOps teams are required to have a reactive mechanism in a place to quickly identify and mitigate the failures. 
+
+- Get an alert whenever there is a sudden spike/ sharp increase in latency of any operation/ API calls from or to my service.
+- Get an alert whenever there is a sudden spike in traffic/ increase in call rate to my service/ API. 
+- Get an alert whenever there is a sudden spike in errors/ increase in error rate for my service. 
+
+## Design
+Currently, we support setting up the rules via a config or via deployment. However, in the UI, we will have a way to list the alert rules and notification configuration. If we closely look at the alerting service, we see that it is simply an anomaly condition from a baseline for a particular metric. So to meet the requirements, we have a **metric anomaly definition** (alert rule) which consists of metric selection and baseline calculation steps. Then we have an **anomaly evaluation engine** which evaluates this rule over a metric (alert rule evaluation engine) and an **alert notification engine** (notification sender) to send the notifications for alerting. Overall architecture diagram now looks like:
+
+![ht-architecture](https://github.com/hypertrace/hypertrace-alert-engine/blob/readme/images/ht-architecture.png)
```

Comment:
```suggestion
![ht-architecture](../images/ht-architecture.png)
```

Reply:
don't use link because this `readme` branch will be deleted once you merge PR and it will fail.